tests/integration/src/test/java/org/apache/pulsar/tests/integration/cli/topicpolicies/SchemaCompatibilityStrategyTest.java

```diff
@@ -19,67 +19,53 @@
 
 package org.apache.pulsar.tests.integration.cli.topicpolicies;
 
-import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.assertEquals;
 import org.apache.pulsar.common.policies.data.SchemaCompatibilityStrategy;
 import org.apache.pulsar.tests.integration.docker.ContainerExecResult;
 import org.apache.pulsar.tests.integration.suites.PulsarTestSuite;
 import org.awaitility.Awaitility;
 import org.testng.annotations.Test;
 
 public class SchemaCompatibilityStrategyTest extends PulsarTestSuite {
-    @Override
-    public void setupCluster() throws Exception {
-        super.setupCluster();
-    }
-
-    @Override
-    public void tearDownCluster() throws Exception {
-        super.tearDownCluster();
-    }
-
     @Test
-    public void testSchemaCompatibilityCmd() throws Exception {
-        String topicName = generateTopicName("",true);
+    public void testSchemaCompatibilityStrategyCmd() throws Exception {
+        String topicName = generateTopicName("test-schema-compatibility-strategy", true);
         pulsarAdmin.topics().createNonPartitionedTopic(topicName);
 
-        Awaitility.await().untilAsserted(()->{
```

Comment:
Why remove the `Awaitility` usage here? I assume this was present because there is a propagation race for the strategy.

Reply:
Only topic policies are asynchronous in Pulsar,  so I remove this`Awaitility`.